### ISSUE TITLE
Suggestion: Add setting to avoid converting pasted tabs into spaces

### ISSUE THREAD

**Issue by sbrl** [ether/etherpad-lite#4823]
* * *

name: Feature request
about: Add setting to avoid converting pasted tabs into spaces
title: 'Add setting to avoid converting pasted tabs into spaces'
labels: enhancement
assignees: 

* * *

**Is your feature request related to a problem? Please describe.**

When I'm drafting a markdown wiki page with friends, it's a frustrating experience when I want to add a table. Typically I use tabs to align text into multiple columns, but etherpad converts them to spaces and breaks everything. 

**Describe the solution you'd like**
I would like a setting to be added that, when enabled, allows tab characters to be entered as normal and causes etherpad to leave all pasted tabs alone.

**Describe alternatives you've considered**
I've looked in the settings already, but didn't find a setting that does this.

**Plugin?**
This sounds like a core text editing feature, rather than a plugin - especially since it's something that etherpad does that I don't want it to.

**Comment by JohnMcLear** [comment]
This is because the notion of a tab is Afaik not part of the HTML spec, so you have either 4 spaces or a margin on an element.

> TL;DR – In HTML, tab is a piece of whitespace equal to four HTML spaces in size.

Can you record a quick video of the user experience and explain how we can improve it?  Also if this issue is specific to tables, would it be more suitable for a plugin/table to support and not be part of core?

**Comment by sbrl** [comment]
Ah, I see. The thing is that I use etherpad to edit Markdown collaboratively on occasion that I then paste into another application, so it's awkward when it translates the tabs I use to intent into spaces.

Hrm, I'm not sure what you're after in terms of me talking about user experience, and I'm really not comfortable with recording my own voice and posting a link to it publicly here.

A plugin for tables would be cool, but not what I'm getting at here.

**Comment by sbrl** [comment]
Excuse me @stale, but I'm not the one who hasn't responded here.
